### ISSUE TITLE
docs.rs build is currently broken due to cmake version

### ISSUE THREAD

**Issue by nmandery** [nmandery/h3ron#46]
The cmake version available in the docs.rs build-env is too old, installing a newer version would require a distro upgrade to ubuntu 22.04. From the build logs:

```
[INFO] [stderr] 
[INFO] [stderr]   --- stderr
[INFO] [stderr]   CMake Error at CMakeLists.txt:15 (cmake_minimum_required):
[INFO] [stderr]     CMake 3.20 or higher is required.  You are running version 3.16.3
[INFO] [stderr] 
```

Related upstream issue: https://github.com/rust-lang/crates-build-env/issues/84

**Comment by nmandery** [comment]
This should be resolved when a #47 is released.

**Comment by nmandery** [comment]
`h3ron` 0.15.1 has been pushed to crates.io and the docs are being build again: https://docs.rs/h3ron/latest/h3ron/

There are no new releases for `h3ron-graph` and `h3ron-ndarray` yet, so the docs of these two will remain broken for a while.
